### ISSUE TITLE
Pretty print test failure output

### ISSUE THREAD

**Issue by danielcompton** [cursive-ide/cursive#1521]
It is quite hard to parse the test failure text when hovering over it with the mouse, which means I need to use the diff almost every time. It could be handy to use something like [humane-test-output](https://github.com/pjstadig/humane-test-output) to prettify the test failure message in the hover text.

**Comment by marco-m** [comment]
Hello, any news? As an alternative, what would be the best way to tell lein to use Humane test output when spawning a REPL outside IDEA/Cursive (that is, a classic "lein repl" in a terminal), and not to use it when spawning a REPL controlled by Cursive? Thanks

**Comment by marco-m** [comment]
Ah, simple and effective, thanks Daniel!
